paddle/legacy/utils/Util.h

```diff
@@ -14,7 +14,9 @@ limitations under the License. */
 
 #pragma once
 
+#ifndef _WIN32
```

Comment:
you modify this file, do that means our legacy code also need to support windows now?

Reply:
Right now the unit test still reply on the legacy part, when I deep into the code part and found porting the legacy part into fluid would be a heavy work, which contains tens of files needs to be modified.

So I simply add some conditional macro to make the test works.